### DISCUSSION TITLE
Is it possible to use a folder for an article?

### DISCUSSION THREAD

**Discussion by zimbatm** [srid/emanote#203] [Q&A] [answered]
For larger articles, it's handy to be able to group all the related assets in a folder, with the article as `index.md` or `README.md`. I can throw some code, pictures, and use [mdsh](https://github.com/zimbatm/mdsh) as a poor man's Jupyter notebook.

I tried putting a index.md or README.md in a folder but it looks like emanote takes it as a sub-page article. And emanote recommend creating a `<folder-name>.md` as the index page. Did I miss a functionality by any chance?

Edit: another advantage is that it makes it easier to copy the article around with all its related assets.

**Comment by srid** [comment, marked as answer]
Presently you would create `<folder-name>.md` along side `<folder-name>/`. The reason for this design decision is that it would allow you to link to the folder as `[[folder-name]]`, and it would work fine with Zettelkasten apps like Obsidian.

A proposal for supporting `index.md` is here: https://github.com/srid/emanote/discussions/107 - and I think it is worth supporting.

**Reply by zimbatm** [reply]
Thanks for letting me know. I understand that it takes a lot of tricky balancing between features and compatibility in this design space.

**Comment by srid** [comment]
Working demo: https://github.com/srid/emanote/pull/512
